python_basics/class03/exercise_6.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[107]:
+
+
+'''
+Edit this file to complete Exercise 6
+'''
+
+def calculation(a, b):
+    '''
+    Write a function calculation() such that it can accept two variables 
+    and calculate the addition and subtraction of it. 
+    It must return both addition and subtraction in a single return call
+
+    Expected output:
+    res = calculation(40, 10)
+    print(res)
+    >>> (50, 30)
+
+    Arguments:
+    a: first number 
+    b: second number
+
+    Returns:
+    sum: sum of two numbers
+    diff: difference of two numbers
+    '''
+
+    # code up your solution here
+    return(a+b, a-b)
+
+
+def triangle_lambda():
+    '''
+    Return a lambda object that takes in a base and height of triangle
+    and computes the area.
+
+    Arguments:
+    None
+
+    Returns:
+    lambda_triangle_area: the lambda
+    '''
+    lambda_triangle_area = lambda b, h: 0.5*b/h
+
+    return lambda_triangle_area
+
+
+def sort_words(hyphen_str):
+    '''
+    Write a Python program that accepts a hyphen-separated sequence of words 
+    as input, and prints the words in a hyphen-separated sequence after 
+    sorting them alphabetically.
+
+    Expected output:
+    sort_words('green-red-yellow-black-white')
+    >>> 'black-green-red-white-yellow'
+
+    Arguments:
+    hyphen_str: input string separated by hyphen
+
+    Returns:
+    sorted_str: string in a hyphen-separated sequence after 
+    sorting them alphabetically
+    '''
+
+    # code up your solution here
+    
+    list2=hyphen_str.split('-')
+    list2 = sorted(list2)
+
+    sorted_str=''
```

Comment:
this works; using `.join` might be a more efficient way

Reply:
got it thanks